lotemplate/classes.py

```diff
@@ -191,6 +217,49 @@ def scan_text(doc) -> dict[str, dict[str, str]]:
 
             return plain_vars | text_fields_vars
 
+        def scan_if(doc) -> None:
+            """
+            scan for if statement. No return. We just verify that there is
+            and endif for each if statement
+            """
+
+            def scan_single_if(local_x_found):
+                """
+                scan for a single if statement
+                """
+                if_statement = IfStatement(local_x_found.getString())
+                position_in_text = len(if_statement.if_string)
+                text = local_x_found.getText()
+                cursor = text.createTextCursorByRange(local_x_found)
+                if not cursor.goRight(1, True):
+                    raise errors.TemplateError(
+                        'no_endif_found',
+                        f"The statement {if_statement} has no endif",
+                        dict_of(if_statement)
+                    )
+                position_in_text = position_in_text + 1
```

Comment:
position_in_text += 1

Reply:
thanks for this syntax I didn't know un python

fixed